src/plugins/orangehrmAuthenticationPlugin/entity/ResetPasswordRequest.php

```diff
@@ -58,6 +58,13 @@ class ResetPasswordRequest
      */
     private string $resetCode;
 
+    /**
+     * @var bool
+     *
+     * @ORM\Column(name="valid", type="boolean")
+     */
+    private bool $valid;
```

Comment:
```suggestion
    private bool $expired;
```
any idea to refactor all wording to `expired`

Reply:
yes expired is more suitable than valid